src/components/pages/DrawingSub/RenderDrawingSub.js

```diff
@@ -35,12 +35,15 @@ export const RenderDrawingSub = props => {
           setModalVisible(false);
         }}
         style={{ fontSize: '1.5rem' }}
-        instructions={modalText || modalInstructions.drawingSub}
+        instructions={modalText}
       />
       <div className="writing-sub-container">
         <Row className="main-row">
-          <p>{props.tasks.story.drawingPrompt}</p>
+          <p>{modalInstructions.drawingSub}</p>
```

Comment:
OH! I think that lorem ipsum in the drawingPrompt was the (poorly designed) seed data for what the student should be "prompted" (or asked) to draw. It might be more useful to change the seed data to full-sentence writing prompts that makes sense, instead of removing it entirely and only displaying the student's submission.

Reply:
@Jahteo That is true, but I am not sure what that text would be about. We already have texts in the redux explaining to the user what they are supposed to do in this step and some tips. Personally I couldn't see any other messages that we'd need to show the users. Am I missing anything?

PS. I am attaching a screenshot of how the UI looks like after the changes in this PR.

<img width="1433" alt="Screen Shot 2021-02-03 at 4 01 47 PM" src="https://user-images.githubusercontent.com/1468463/106825248-54fd7d80-6639-11eb-8c42-d598bf753215.png">